index.html

```diff
@@ -43,3 +43,10 @@ <h1><img src="/assets/logo.png" id="logo" alt="Sway" /></h1>
     </p>
   </div>
 </div>
+<div class="row" style="margin-top: 1rem;">
+  <div class="col-md-12">
+    <p>
+      NOTE: The IRC channel #sway has been moved from Freenode.net to <a href="https://libera.chat">Libera Chat</a>.
```

Comment:
```suggestion
      NOTE: The IRC channel #sway has been moved from freenode to <a href="https://libera.chat">Libera Chat</a>.
```
I believe this is the official capitalization and name. Also not really sure if this notice is necessary.

Reply:
I am also not sure, if this is needed or not. I am not against dropping it.

I have added this only to mention the name of the channel.
With the old link to the webchat, the correct channel was part of the link.
The new link opens just the landing page of Libera Chat. New users might not know the correct channel name.
Another possibility is to change the link in the other file to something like "Support Chat (channel #sway) >>"